app/assets/stylesheets/components/_base.sass

```diff
@@ -8,3 +8,6 @@
 @import 'main_layout'
 @import 'navbar'
 @import 'notification'
+@import 'user_profile_layout'
```

Comment:
Please try not to couple the component name to the content its applied in a certain moment of the app. make it generic